src/config.py

```diff
@@ -10,22 +10,19 @@
 load_dotenv("api_keys.env")
 
 
-def get_env_variable(var_name: str,
-                     default: T = None, allow_empty: bool = False, cast_to: Type[T] = str) -> T:
+def get_env_variable(var_name: str, default: str = None, allow_empty: bool = False) -> str:
     """
-    Reads an environment variable, with settings to allow defaults, empty values, and type casting
-    To read a boolean EXAMPLE_ENV_VAR=False use get_env_variable("EXAMPLE_ENV_VAR", cast_to=bool)
+    Reads a string environment variable, with settings to allow defaults, empty values.
+    To read a boolean user get_bool_env_variable
 
     Parameters
     ----------
     var_name : str
         The name of the environment variable to retrieve.
     default : T = None
-        Default return value if the environment variable does not exist. Doesn't override empty string vars.
+        Default return value if the environment variable does not exist.
     allow_empty : bool
-        If False then a KeyError will be raised if the environment variable is empty.
-    cast_to : Type[T]
-        The type to cast to e.g. str, int, or bool
+        If False then a ValueError will be raised if the environment variable is empty.
 
     Returns
```

Comment:
Update the Return to be accurate.

Example: https://github.com/LukeParky/NewZeaLiDAR/blob/6a83fad648d0700ae18a75afb056e2d8b5509cad/newzealidar/env_var.py